edgePy/_DGEList.py

```diff
@@ -196,7 +196,7 @@ def rpkm(
         gene_lengths: Mapping,
         log: bool = False,
         prior_count: float = PRIOR_COUNT
-    ) -> 'DGEList':
+    ) -> None:
```

Comment:
FYI these were written as strings to combat forward referencing of a type which is impossible in Python.

PEP484 has the topic:

https://www.python.org/dev/peps/pep-0484/#forward-references